src/logistration/tests/RegistrationPage.test.jsx

```diff
@@ -66,4 +71,40 @@ describe('./RegistrationPage.js', () => {
     renderer.create(reduxWrapper(<IntlRegistrationPage />));
     expect(window.location.href).toBe(dasboardUrl);
   });
+
+  it('should show error message on 409', () => {
+    const windowSpy = jest.spyOn(global, 'window', 'get');
```

Comment:
just for clarification, why are we mocking window object?

Reply:
For scrollTo function.